src/App.tsx

```diff
@@ -0,0 +1,21 @@
+import NavBar from "./components/NavBar/NavBar";
+import Card from "./components/Card/Card";
+import { data } from "./data";
+import React, { useState } from "react";
+import TravelForm from "./components/TravelForm/TravelForm";
+
+export default function App(): JSX.Element {
+    const [travelInfo, setTravelInfo] = useState(data);
```

Comment:
you may want to add the <TripEntry[]> type on this guy. To soecifically tell it what type it should expect.
just pop it between the useState and the (data)